docs/components/text-control-with-affixes.md

```diff
@@ -0,0 +1,71 @@
+Text Control With Affixes
```

Comment:
These docs files are actually generated from inline docs in the component, not hand-written – I've [added actual instructions for this,](https://github.com/woocommerce/wc-admin/blob/edf68545802824cf1cf9bc4d3e803d5cfedf3a54/docs/documentation.md) can you update the `TextControlWithAffixes` file with this as inline docs?

Reply:
Done in 5e2d473.